### PR TITLE
Dingo integration

### PR DESCRIPTION
**Pull request by rhysfaultless-cpr**

### CODE REVIEW

File: docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  <img
+    src="/img/robot_images/dingo_images/dingo_pacs_3.png"
+    width="300"
+  />
+</center>
+
+:::note
+
+We suggest you to remove the 3rd and 5th rows of screws from the front of the Dingo-D or Dingo-O.
+You can choose different locations to suit your application,but this will require custom URDF software work.
```
Comment (jhiggins-cpr) [inline docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx]: missing space after "application,"

File: docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+| Kit, Riser Full—Dingo, 200 |  027033  |
+| Kit, Riser Full—Dingo, 300 |  027034  |
+| Kit, Riser Full—Dingo, 400 |  027035  |
+| Kit, Riser Full—Dingo, 500 |  027036  |
+
+## Parts List
+
+| ID  | Description                                         |                         CPR item                         | Quantity |
+| :-: | :-------------------------------------------------- | :------------------------------------------------------: | :------: |
+|  1  | Plate—Integration, 510 X 430                        |   [026909](/assets/pdf/clearpath_robotics_026909.pdf)    |    1     |
```
Comment (jhiggins-cpr) [inline docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx:27]: Is it the same riser plate for Dingo-D and Dingo-O? I was initially assuming that they were different (and that the one for Dingo-O would be larger), but maybe that is too many parts to carry in inventory.

File: docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx (hunk 1 above)
Comment (rhysfaultless-cpr) [reply docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx]: Fixed

File: docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx (hunk 2 above)
Comment (rhysfaultless-cpr) [reply docs/robots/dingo/integration_dingo/kits/riser_full_dingo.mdx:27]: We have a common item for Dingo-D and Dingo-O. The plate is 6 PACS positions long, to match the Dingo-Differential's length. The Dingo-O is 7 PACS positions long.

You can see this difference on https://build.clearpathrobotics.com/

**Pull request merged**